substrate/frame/vesting/src/tests.rs

```diff
@@ -78,6 +79,7 @@ fn check_vesting_status() {
 
 		System::set_block_number(10);
 		assert_eq!(System::block_number(), 10);
+		assert_ok!(Vesting::do_try_state());
```

Comment:
We can avoid calling the try state checks all over the tests by a small refactor of the externalities builder, as mentioned in another comment. That should suffice in most cases, keeps the code cleaner and in the future the developer doesn't need to remember to include a call to the try state.

Reply:
Yes, the final test case `play_out_all_schedules`, would require sprinkling `do_try_state`.